source/Government.cpp

```diff
@@ -258,7 +258,17 @@ void Government::Load(const DataNode &node)
 		else if(key == "language")
 			language = child.Token(valueIndex);
 		else if(key == "raid")
-			raidFleet = GameData::Fleets().Get(child.Token(valueIndex));
+		{
+			if(!add)
+				raidFleets.clear();
```

Comment:
If I specify two raids in a single system and the second one shouldn't overwrite the first one.

Reply:
It shouldnt? Mb I got events wrong
You can just define them all under a node of raid if you want two of them

Reply:
oh yeah I see what you mean